### DISCUSSION TITLE
How to add store with multiple products on one page

### DISCUSSION THREAD

**Discussion by LetThatSinkIn** [jef/streetmerchant#3131] [Help]
Hello,

I was thinking about trying to add a store (Canakit) that is perpetually sold out of Raspberry Pi's... but I'm hitting a little bit of a roadblock because they list multiple variations of the product (varying levels of RAM) on each page.  

https://www.canakit.com/raspberry-pi-4-starter-kit.html

I don't see any button selectors when I inspect the page, perhaps because it's sold out at the moment?   Is it even possible/practical to add this considering the page layout?

Thanks

**Comment by Mr-Tech-13** [comment]
Add this and fill in the relevant details
{
  brand : '',
  labels : {
    inStock: {
        container: '',
        text: [''],
             },
            },
         model: '',
         series: '',
         url: '',
},

**Reply by Mr-Tech-13** [reply]
This is copied and pasted for each model you add
